composer.json

```diff
@@ -18,12 +19,13 @@
         }
     ],
     "require": {
-        "php": "^7.0 || ^5.6",
-        "cakephp/cakephp": "^3.7",
-        "wyrihaximus/html-compress": "^2.0 || ^1.0"
+        "php": ">=5.5.9",
```

Comment:
Can you also revert this, the current version targeting also allows PHP8, 9, etc. And we don't know yet if those are compatible. Rather lock them down to 5 and 7 until this is tested on the next version and then release a new version with the changes required for 8.